cmd/createcluster.go

```diff
@@ -70,6 +71,9 @@ type clusterConfig struct {
 	SplitKeysDir string
 
 	InsecureKeys bool
+
+	LaunchpadAPIAddr string
```

Comment:
isn't this `ObolAPIAddr` now?

Reply:
Going for PublishAddr in line with the flag